rest-service/manager_rest/test/endpoints/test_deployment_groups.py

```diff
@@ -359,6 +359,36 @@ def test_remove_from_filters(self):
         group = self.client.deployment_groups.get('group1')
         assert group.deployment_ids == []
 
+    def test_set_labels(self):
+        """Shrink a group providing filter_id"""
+        labels = [{'label1': 'value1'}]
+        updated_labels = [{'label1': 'value2'}, {'label2': 'value3'}]
+        group = self.client.deployment_groups.put(
+            'group1',
+            labels=labels,
+        )
+        self.assert_resource_labels(group.labels, labels)
+        group = self.client.deployment_groups.put(
+            'group1',
+            labels=updated_labels,
+        )
+        self.assert_resource_labels(group.labels, updated_labels)
+
+    def test_group_labels_for_deployments(self):
+        """Shrink a group providing filter_id"""
+        group = self.client.deployment_groups.put(
+            'group1',
+            labels=[{'label1': 'value1'}, {'label2': 'value2'}],
+            blueprint_id='blueprint',
+            new_deployments=[{
+                'labels': [{'label1': 'value1'}, {'label3': 'value3'}]
+            }]
+        )
+        dep_id = group.deployment_ids[0]
+        dep = self.sm.get(models.Deployment, dep_id)
+        self.create_deployment_environment(dep)
+        dep = self.sm.refresh(dep)
```

Comment:
is this test about making sure nothing breaks along the way or… ?

Reply:
no, i forgot to add asserts ;P